diagnostic.md

```diff
@@ -35,19 +35,19 @@ Open up this file in Atom, and write your answers below (where indicated).
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+This is an absolute path because it begins with a /, which indicates the root directory is the first level of this path.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+We would use a relative path to refer to the font because we do not have the file for the font downloaded onto our machines.  It would have to reference a website instead.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
```

Comment:
Relative. When it's within a working directory, you use the relative path to access it. The file is relative to the directory you're working in, where as absolute would start at the root `/Users/etc.`